.github/workflows/test.yml

```diff
@@ -220,6 +220,10 @@ jobs:
       - name: Set permissions
         run: chmod +x ./saucectl
 
-      - name: Saucectl RUN
+      - name: Saucectl RUN - Config Driven
+        run: |
+          ./saucectl run -c .sauce/xcuitest.yml --timeout 10m
+
+      - name: Saucectl RUN - CLI Driven
         run: |
-          ./saucectl run -c .sauce/xcuitest.yml --timeout 10m
\ No newline at end of file
+          ./saucectl run xcuitest --name "cli driven" --app ./apps/SauceLabs.Mobile.Sample.XCUITest.App.ipa --testApp ./apps/SwagLabsMobileAppUITests-Runner.ipa --device name="iPhone.*" --timeout 10m
```

Comment:
Just ran this command locally. It says
```
Error: failed to locate project configuration: open /Users/tianfeng/code/go/src/saucectl/.sauce/config.yml: no such file or directory
Usage:
  saucectl run xcuitest [flags]
...
```
I think it's missng `-c .sauce/xcuitest_light.xml`

Reply:
And also should be
```
--app ./tests/e2e/xcuitest/SauceLabs.Mobile.Sample.XCUITest.App.ipa \
--testApp ./tests/e2e/xcuitest/SwagLabsMobileAppUITests-Runner.ipa
```

Reply:
Copy paste strikes again 😅